Process/ReadProcess.c

```diff
@@ -0,0 +1,75 @@
+#include"ReadProcess.h"
+void get_job(char * path)
```

Comment:
We need to follow a same strategy for naming convention. For example: Upper Camel Case. (https://en.wikipedia.org/wiki/Camel_case)

Reply:
I done